assignments/function-conversion.js

```diff
@@ -5,26 +5,35 @@
 // };
 // myFunction();
 
+// let myFunction = () =>  console.log('Function was invoked!');
+// myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
-
+// let anotherFunction = (param) =>  param;
+// console.log(anotherFunction("Example"));
```

Comment:
when there is only one param the parenthesis can be omitted like so: let anotherFunction = param => param;